backend/core/src/shared/units-transformations.ts

```diff
@@ -332,9 +331,10 @@ export const summarizeUnitsByTypeAndRent = (units: Units): UnitSummary[] => {
     const finalSummary = unitMap[key].reduce((summary, unit, index) => {
       return getUnitsSummary(unit, index === 0 ? null : summary)
     }, {} as UnitSummary)
-    finalSummary.totalAvailable = unitMap[key].filter(
-      (unit) => unit.status === UnitStatus.available
-    ).length
+    // TODO: YL
+    // finalSummary.totalAvailable = unitMap[key].filter(
+    //   (unit) => unit.status === UnitStatus.available
+    // ).length
```

Comment:
We should check with @kathyccheng on this. I think it makes sense to either update this to be total units regardless of availability or to only show the count if the listing is marked as available.

Reply:
Hey Sean I agree, but this is actually handled in a different ticket/pr so once all of them are merged in then this gets resolved